include/logger.h

```diff
@@ -41,14 +41,18 @@ class CoreExport FileWriter
 	 */
 	FILE* log;
 
+	/** The number of write operations after which we should flush.
+	 */
+	int flush;
```

Comment:
It may seem nice to use an `unsigned int` to indicate that something can never be negative, but it has consequences when doing arithmetic with a regular `int`. Such is the case further down with `if (++writeops % flush == 0)`.  I'm not sure if there is any impact in this particular scenario, but such an example can be seen [here](http://stackoverflow.com/questions/24745998/c-modulo-of-an-unsigned-integer).